server/src/resolvers/resourceRequest.ts

```diff
@@ -113,6 +127,50 @@ export class ResourceRequestResolver {
         ],
       };
     }
+
+    const managerStatus = ["Open", "Cancelled", "On Hold"];
+    const tpdStatus = [
+      "Open",
+      "Cancelled",
+      "On Hold",
+      "Moved",
+      "Pending Hiring Request",
+      "Hired",
+      "Pending Outsourcing Request",
+      "Outsourced",
+      "Over allocated",
+    ];
+
+    if (
+      (req.session!.userRole === UserRole.MANAGER &&
```

Comment:
const userRole = req.session?.userRole;
then use it in the if condition.

use `R` instead of `UserRole`